src/pages/commits/ActiveCommits/ChartToggle.tsx

```diff
@@ -31,14 +31,14 @@ export const ChartToggle: React.FC<{
             id="chart-radio-absolute"
             value={ChartTypes.Absolute}
           >
-            <Label htmlFor="chart-radio-absolute">Absolute Number</Label>
+            <b>Absolute Number</b>
           </Radio>
           <Radio
             data-cy="cy-chart-percent-radio"
             id="chart-radio-percent"
             value={ChartTypes.Percentage}
           >
-            <Label htmlFor="chart-radio-percent">Percentage</Label>
+            <b>Percentage</b>
           </Radio>
```

Comment:
Padding is still off; I think for some reason leafygreen adds an `8px margin-top` to the radio buttons??

this `padding` will fix it:
<img width="669" alt="Screen Shot 2022-02-08 at 5 32 31 PM" src="https://user-images.githubusercontent.com/47064971/153087388-85ba0a66-7b45-4239-8fcf-d8c5f41ff270.png">